proto_patch.sed

```diff
@@ -50,4 +50,13 @@ s+repeated Exemplar exemplars = \(.*\);+repeated Exemplar exemplars = \1\
   [ (gogoproto.nullable) = false ];+g
 
 s+Buckets \(.*\)tive = \(.*\);+Buckets \1tive = \2\
-  [ (gogoproto.nullable) = false ];+g
\ No newline at end of file
+  [ (gogoproto.nullable) = false ];+g
+
+s+optional double \(.*\);+OptionalDouble \1\
+  [\
+    (gogoproto.customtype) = "go.opentelemetry.io/collector/model/internal/data.OptionalDouble"\
```

Comment:
I think we can have "not nullable" and keep a boolean in OptionalDouble about "has_value"?

Reply:
i agree, updated this, ptal